go/cmd/vttestserver/cli/main.go

```diff
@@ -219,25 +221,44 @@ func New() (cmd *cobra.Command) {
 	cmd.Flags().StringVar(&config.ExternalTopoGlobalRoot, "external_topo_global_root", "", "the path of the global topology data in the global topology server for vtcombo process")
 
 	cmd.Flags().DurationVar(&config.VtgateTabletRefreshInterval, "tablet_refresh_interval", 10*time.Second, "Interval at which vtgate refreshes tablet information from topology server.")
+
+	cmd.Flags().BoolVar(&doCreateTCPUser, "initialize_with_vt_dba_tcp", false, "If this flag is enabled, MySQL will be initialized with an additional user named vt_dba_tcp, who will have access via TCP/IP connection.")
```

Comment:
We have agreed to use hyphens in the new flags that we add to Vitess. So could you please rename the flag to `initialize-with-vt-dba-tcp`

Reply:
I renamed the flag [6f5e853](https://github.com/vitessio/vitess/pull/15354/commits/6f5e853d921ba4a14909beebd5e1978a50460c19)